Code/ThirdParty/LivePP/init.cmake

```diff
@@ -0,0 +1,31 @@
+# LIVE++ Utils
+# NOTES: To not use github storage, you can download the Live++ SDK from the official website and put it in the same directory as this file.
+
+if(EZ_CMAKE_PLATFORM_WINDOWS)
+    set(EZ_SUPPORT_LIVEPP ON CACHE BOOL "Enable Live++ support for Ez")
```

Comment:
Should be named EZ_3RDPARTY_LIVEPP_SUPPORT to be in line with other 3rd party integrations.

Also should be off by default on all platforms, but can be automatically set to ON, when the necessary binaries are copied here by the user.

Reply:
@jankrassnigg Can this be marked as resolved on your end since it is fixed in newer commits to the PR